docs/tutorial/part-two/index.md

```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
 
 -->
 
-Welcome to part two of the Gatsby tutorial!
+Selamat datang di bagian dua dari tutorial Gatsby!
```

Comment:
Bagian kedua, i think its better use that word

Reply:
i agree